specs/svg-native/index.bs

```diff
@@ -128,128 +190,88 @@ Only the following units are supported:
+The <{marker}> element is not supported by SVG Native, nor are the 'marker', 'marker-start', 'marker-mid', and 'marker-end' properties.
+<!-- https://www.w3.org/2019/07/15-svg-minutes.html -->
 
-Color {#color}
-==============
+The ''context-fill'' and ''context-stroke'' values are not supported by SVG Native.
 
-Delete section 12.2: The 'color' property.
+Note: Authors wishing to use a contextual color from the environment may use ''currentColor''.
+<!-- https://www.w3.org/2019/06/17-svg-minutes.html -->
 
-Note: In some environments, the ''currentColor'' keyword is used to provide surrounding context to the SVG graphic. For example, in a color font glyph, ''currentColor'' may represent the text color of the surrounding line. Therefore, support of ''currentColor'' is required, but support of the 'color' property to change ''currentColor'' is not supported.
+The <{color}> property is not supported by SVG Native.
+<!-- https://www.w3.org/2019/06/17-svg-minutes.html -->
 
-Delete section 12.3: Color profile descriptions.
+Note: The absence of <{color}> makes it impossible to change the value of ''currentColor'' from its initial value.
 
-Note: This specification will likely adopt [[!SVG2]]'s syntax for specifying colors outside of sRGB.
+The 'vector-effect', 'paint-order', 'color-interpolation', and 'will-change' attributs are not supported by SVG Native.
 
-ISSUE: What should we do about non-sRGB color spaces?
+All color interpolation occurs in the sRGB color space.
 
 Gradients and Patterns {#pservers}
 ==================================
 
-The following attributes are forbidden:
-- 'gradientUnits'
-- 'xlink:href' on <{linearGradient}> or <{radialGradient}>
+Only the ''userSpaceOnUse'' value of 'gradientUnits' is supported. According to [[!SVG2]], the default value of 'gradientUnits' is ''objectBoundingBox'', which isn't supported. This means that all <{gradient}> elements must include <code>gradientUnits="userSpaceOnUse"</code> to be valid.
+<!-- https://www.w3.org/2019/11/04-svg-minutes.html -->
 
-Delete section 13.3: Patterns.
+<!-- href is supported on gradients. https://www.w3.org/2019/07/15-svg-minutes.html -->
 
-Clipping, Masking, and Compositing {#masking}
-=============================================
+The <{pattern}> element is not supported by SVG Native.
+<!-- https://www.w3.org/2019/07/15-svg-minutes.html -->
 
-Delete section 14.4: Masking.
+Scripting and Interactivity {#interact}
+=======================================
 
-The following attributes are forbidden:
-- 'clipPathUnits'
-- 'clip'
-
-The ''scroll'' value on the 'overflow' property behaves identically to 'hidden'.
-
-The 'clip-path' attribute must not be present on a <{clipPath}> element.
-
-Every <{clipPath}> element must have either exactly zero or exactly one child.
-
-The ''scroll'' and the ''auto'' values of the 'overflow' property are forbidden.
-
-Filter Effects {#filters}
```

Comment:
I know that filters aren't defined in SVG anymore, but the `filter` property is listed as a required CSS feature, so you will need to still explicitly exclude it.

And, since the filter elements are still SVG namespace elements, and you have explicitly talked about clipPath and mask elements, maybe want to keep something about no filter support.

(Unless I missed a decision where we decided to *allow* filters…?)

Reply:
I believe I excluded it in the "SVG Modules" modules section above.

> No other SVG modules are supported by SVG Native. This includes:
> - <a href="https://www.w3.org/TR/filter-effects-1/">CSS Filter Effects</a>